docs/pubsub-usage.rst

```diff
@@ -20,24 +32,27 @@ Create a new topic for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
    >>> topic.create()  # API request
 
-Create a new topic for an explicit project:
+Create a new topic for an explicit client:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name', project='my.project')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = Topic('topic_name', client=client)
```

Comment:
I 100% agree that this should exist (I absolutely see the uses for it), but should we explain that in the full usage here? I would more expect that we'd recommend something like...

```python
client = pubsub.Client()
client2 = pubsub.Client.with_service_account_json('key.json')

topic = client.topic('name')
explicit_topic = client2.topic('name')
```

If someone wants to look at the method and see "oh cool, I can pass in an explicit client", that's fine. But I'd rather we recommend that you create a separate client and then use that for your interaction -- no?